lib/mappers/bhl_mods.py

```diff
@@ -42,17 +41,14 @@ def map_description(self):
         if exists(self.provider_data, prop):
             description = []
             for s in iterify(getprop(self.provider_data, prop)):
-                try:
-                    if s.get("type") == "content":
-                        description.append(s.get("#text"))
-                except:
-                    logger.error("Error getting note/type from record %s" %
-                                 self.provider_data["_id"])
+                if "type" in s and s.get("type") == "content":
+                    description.append(s.get("#text"))
 
-            self.update_source_resource({"description": description})
+            if description:
+                self.update_source_resource({"description": description})
 
-    # OAI MODS Mapper is blowing up on this method because of arg c
-    # ount issues, even though we handle it below
+    # OAI MODS Mapper is blowing up on this method because of arg
+    # count issues,
```

Comment:
I see what's happing with `format` now. I was looking at the wrong row of the crosswalk spreadsheet. I'm seeing how format is mapped by `map_format_and_spec_type`.

Both format and type are confusing because of the way they're stuck out in the spreadsheet, but I'll leave it up to @guegueng in the QA phase to say what to do with these fields.